src/client.ts

```diff
@@ -601,6 +602,17 @@ export class GenesysCloudWebrtcSdk extends (EventEmitter as { new(): StrictEvent
       });
     });
   }
+
+  /**
+   * Start a softphone session with the given peer or peers.
+   *  `initialize()` must be called first.
+   *
+   * @param softphoneParams participant information for initiating a softphone session. See IStartSoftphoneSessionParams for more details.
+   */
+  async startSoftphoneSession (softphoneParams: IStartSoftphoneSessionParams): Promise<any> {
```

Comment:
👍🏼  function comment looks good too

Reply:
Is there a way to type the response? I am not sure how consistent the data is that the API returns.

Reply:
Its been added!